src/mainwindow.cpp

```diff
@@ -5124,7 +5124,6 @@ void MainWindow::updateCurrentFolderTooltip() {
  */
 void MainWindow::openSettingsDialog(int page, bool openScriptRepository) {
     QPointer<SettingsDialog> settingsDialog = new SettingsDialog(page, this);
-    settingsDialog->readSettings();
```

Comment:
Hm, may cause regressions...

Reply:
Ah, never mind, that was redundant since a new instance of the dialog is used every time.

Reply:
yup